CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 1.0.0 - 2018-02-28
```

Comment:
Version number should be 1.0.1 right? I thought the 1.0.0 release failed?

Reply:
yeah there's a tag for 1.0.1. Also the pom point to 1.0.1-SNAPSHOT that should be 1.0.2-SNAPSHOT